<launch>
<arg name="mav_name" default="iris"/>
  <arg name="fcu_url" default="udp://:14540@127.0.0.1:14557"/>
  <arg name="gcs_url" default="" />
  <arg name="tgt_system" default="1" />
  <arg name="tgt_component" default="1" />
  <arg name="command_input" default="2" />
  <arg name="gazebo_simulation" default="true" />
  <arg name="visualization" default="true"/>
  <arg name="log_output" default="screen" />
  <arg name="fcu_protocol" default="v2.0" />
  <arg name="respawn_mavros" default="false" />
  <node pkg="offb" type="offb_attitude" name="offb_attitude" output="screen">
          <param name="controller_px" value="2.0" />
          <param name="controller_py" value="2.0" />
          <param name="controller_pz" value="9.0" />
          <param name="controller_dx" value="0" />
          <param name="controller_dy" value="0" />
	  <param name="controller_dz" value="4.0" />
          <param name="controller_ix" value="0" />
          <param name="controller_iy" value="0" />
          <param name="controller_iz" value="0.07" />
	  <param name="target_x" value="3.0" />
	  <param name="target_y" value="-1" />
          <param name="target_z" value="2" />	

  </node>
<include file="$(find mavros)/launch/node.launch">
      <arg name="pluginlists_yaml" value="$(find mavros)/launch/px4_pluginlists.yaml" />
      <arg name="config_yaml" value="$(find mavros)/launch/px4_config.yaml" />

      <arg name="fcu_url" value="$(arg fcu_url)" />
      <arg name="gcs_url" value="$(arg gcs_url)" />
      <arg name="tgt_system" value="$(arg tgt_system)" />
      <arg name="tgt_component" value="$(arg tgt_component)" />
      <arg name="log_output" value="$(arg log_output)" />
      <arg name="fcu_protocol" value="$(arg fcu_protocol)" />
      <arg name="respawn_mavros" default="$(arg respawn_mavros)" />
  </include>

</launch>
